src/SignTool/SignTool/Program.cs

```diff
@@ -159,6 +160,10 @@ internal static void PrintUsage()
                         test = true;
                         i++;
                         break;
+                    case "-testSign":
+                        testSign = true;
+                        i++;
+                        break;
```

Comment:
The usage string may need to be fixed. I see the following usage string (which does not include -testSign) -
```
shell> "C:\Users\******\.nuget\packages\roslyntools.microsoft.signtool\0.3.4-beta\tools\SignTool.exe"
Need a value for outputPath
SignTool.exe [-test] [-intermediateOutputPath <path>] [-msbuildPath <path>] [-nugetPackagesPath <path>] [-config <path>] outputPath
```

Reply:
There is also a problem with case sensitivity. Specifying -testSign results in the following unrecognized option error (probably because you are comparing the already lowercased string to a string where the letter S is uppercase.

```
shell> "C:\Users\******\.nuget\packages\roslyntools.microsoft.signtool\0.3.4-beta\tools\SignTool.exe" -testSign -intermediateOutputPath 'C:\Users\gnamboo\Desktop\testimpact\artifacts\Debug\obj\' -msbuildPath 'C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\MSBuild\15.0\Bin\MSBuild.exe' -nugetPackagesPath 'C:\Users\gnamboo\.nuget\packages\' -config 'C:\Users\gnamboo\Desktop\testimpact\build\Signing\BatchSignData.json' 'C:\Users\gnamboo\Desktop\testimpact\artifacts\Debug\bin\'
Unrecognized option -testsign
SignTool.exe [-test] [-intermediateOutputPath <path>] [-msbuildPath <path>] [-nugetPackagesPath <path>] [-config <path>] outputPath

test: Run tool without actually modifying any state.
outputPath: Directory containing the binaries.
intermediateOutputPath: Directory containing intermediate output.  Default is (outputpath\..\Obj).
nugetPackagesPath: Path containing downloaded NuGet packages.
msbuildPath: Path to MSBuild.exe to use as signing mechanism.
config: Path to SignToolData.json. Default build\config\SignToolData.json.
```

Reply:
@jaredpar I have submitted fixes for the above issues. Could you please republish the nuget?